### PR TITLE
The first version of Istio Auth FAQ

### PR DESCRIPTION
**Pull request by wattli**

### CODE REVIEW

**Comment by ZackButcher** [comment]
Release managers don't need to review doc changes btw - they can just go in after a normal review by a team member / subject area expert.

**Comment by rshriram** [comment]
I know. It's just to smooth out the tones and presentation formats that we asked folks cc me/frank/Rachel

File: _docs/tasks/security/faq.md [hunk 1]
```diff
@@ -9,3 +9,39 @@ type: markdown
 ---
 {% include home.html %}
 
+This page summarizes frequently asked questions and the team can be reached via slack (istio.slack.com
+ #auth channel) or google groups (https://groups.google.com/forum/#!forum/istio-users).
```
Comment (rshriram) [inline _docs/tasks/security/faq.md]: Remove mention of slack channel

File: _docs/tasks/security/faq.md [hunk 2]
```diff
@@ -9,3 +9,39 @@ type: markdown
...
+
+### Can a service with Istio Auth enabled communicate a service without Istio?
+
+Currently it is not well supported. But we do have plan to support this in the near future.
+
+### Can I enable Istio Auth with some services while disable others in the same cluster?
+
+No, you cannot for now. Currently we only support cluster-wise Auth enable/disable. It is
```
Comment (rshriram) [inline _docs/tasks/security/faq.md]: mesh-wide auth enable/disable.

Comment (rshriram) [reply _docs/tasks/security/faq.md]: Remove the no you cannot

File: _docs/tasks/security/faq.md [hunk 3]
```diff
@@ -9,3 +9,39 @@ type: markdown
...
+a high priority action item for us to support per-service auth.
+
+### How can I use Kubernetes liveness and readiness to for service health check with Istio Auth enabled?
+
```
Comment (rshriram) [inline _docs/tasks/security/faq.md]: remove to

**Comment by ZackButcher** [comment]
Totally, just wanted to be sure they knew I'm not blocking anything at least.

File: _docs/tasks/security/faq.md [hunk 4]
```diff
@@ -9,3 +9,39 @@ type: markdown
...
+If Istio Auth is enabled, http and tcp health check from kubelet will not work since they do not have
+Istio Auth issued certs. A workaround is to use command option for health check, e.g., one can install
```
Comment (rshriram) [inline _docs/tasks/security/faq.md]: A temporary workaround

File: _docs/tasks/security/faq.md [hunk 5]
```diff
@@ -9,3 +9,39 @@ type: markdown
...
+curl in the service pod and curl itself within the pod. Moreover, this is a temporary workaround. The
+Auth team is actively working on a real solution.
```
Comment (rshriram) [inline _docs/tasks/security/faq.md]: Say the istio team

File: _docs/tasks/security/faq.md [hunk 6]
```diff
@@ -9,3 +9,39 @@ type: markdown
...
+
+### How can I access Kubernetes ApiServer with Auth enabled?
+
+It will not work from service container since the traffic will be intercepted by Istio proxy, and
+ApiServer cannot recognize Istio Auth issued certs. You can either turn off Istio Auth or access
+ApiServer in the proxy container.
+
+To ssh into the proxy container:
```
Comment (rshriram) [inline _docs/tasks/security/faq.md]: Remove this. Proxy container is off limits for end users

**Comment by wattli** [comment]
Great, somehow I can't add Rachel to the reviewer. Can someone help? Thanks.

File: _docs/tasks/security/faq.md (hunk 4 above)
Comment (myidpt) [reply _docs/tasks/security/faq.md]: You can remove "Moreover, this is a temporary workaround." as well.

File: _docs/tasks/security/faq.md (hunk 6 above)
Comment (wattli) [reply _docs/tasks/security/faq.md]: done

File: _docs/tasks/security/faq.md (hunk 5 above)
Comment (wattli) [reply _docs/tasks/security/faq.md]: done

File: _docs/tasks/security/faq.md (hunk 4 above)
Comment (wattli) [reply _docs/tasks/security/faq.md]: done

File: _docs/tasks/security/faq.md (hunk 3 above)
Comment (wattli) [reply _docs/tasks/security/faq.md]: done

File: _docs/tasks/security/faq.md (hunk 2 above)
Comment (wattli) [reply _docs/tasks/security/faq.md]: done

File: _docs/tasks/security/faq.md (hunk 1 above)
Comment (wattli) [reply _docs/tasks/security/faq.md]: done

**Pull request merged**